guide/sapphire/args.md

```diff
@@ -0,0 +1,163 @@
+
+Sometimes when using commands, you may want to get data from the user and change the response accordingly. In this section, you'll create a command that pulls a string from the message and says it back to the user!
+
+## String arguments
+
+A `string` argument is simply the text after the command name and prefix. For example: `@bot say Hello!` would cause the argument to be `Hello!`, although multiple words (e.g. `Hello there!`) are separated in different parameters. We'll look into how to handle those later.
+
+First, go into your `commands` folder and make a new file called `say.js`. Once you have it, set up your command class just like the one in the `ping` command.
+
+```js
+const { Command } = require('@sapphire/framework');
+
+module.exports = class SayCommand extends Command {
+	constructor(context) {
+		super(context, {
+			aliases: ['parrot', 'copy'],
+			description: 'Replies with the text you provide',
+		});
+	}
+
+	run(message, args) {
+		// ...
+	}
+};
+```
+
+We have a new parameter in the `run` method: `args`! This is a structure that abstracts all the parameter parsing and resolution.
+
+```js {5}
+module.exports = class SayCommand extends Command {
+	// ...
+
+	async run(message, args) {
+		const text = await args.pick('string');
+		await message.channel.send(text);
+	}
+};
+```
+
+The way Sapphire's arguments work, is by "consuming" the parameters one by one on success, similar to how an iterator works. We'll go into more details and examples with another command.
+
+In this command, we're using `args.pick` to read a single parameter from the user's input. The input type is `string`, which means it'll pick the current parameter and return it as-is.
+
+If you run this with `@bot say Hello there!`, it'll send `Hello` back. What happened!? As I said before, each word is a parameter, so `Hello there!` is converted to `["Hello", "there!"]`, and `args.pick` reads only one. But fear not! There are alternatives:
+
+- **Quoted parameters**:
+  If instead of sending `@bot say Hello there!`, we sent `@bot say "Hello there!"` or `@bot say “Hello there!”`, wrapping the content with quotes, the entire content inside the quotes will be read as a single parameter (without the quotes, of course).
+
+- **`args.rest` as-is**:
+  ```js {5,10}
+  module.exports = class SayCommand extends Command {
+  	constructor(context) {
+  		super(context, {
+  			// ...
+  			quotes: [],
+  		});
+  	}
+  
+  	async run(message, args) {
+  		const text = await args.rest('string');
+  		await message.channel.send(text);
+  	}
+  };
+  ```
+
+  By default, `Command` defines 3 pairs of valid quotes:
+
+  - `"` and `"` (double quotes)
+  - `“` and `”` (iOS smart quotes)
+  - `「` and `」` (CJK corner brackets)
+
+  This is troublesome if we want to send quotes in the content, as those would be excluded from `rest`, so we have set quotes to an empty array. This way, when you do `@bot say Hello "there!"`, your bot will send `Hello "there!"`.
```

Comment:
please put "quotes" in backticks -- to show it's "code" (i.e. it's a field of the class)

Reply:
What do you mean?

Reply:
"This is troublesome if we want to send quotes in the content" -> " This is troublesome if we want to send `quotes` in the content"